### PR TITLE
Update CODEOWNERS to better reflect reality

### PR DESCRIPTION
**Pull request by dopplershift**

### CODE REVIEW

**Comment by dopplershift** [comment]
Looks like this one is having the weird "leave the team assigned" issue. I wonder if it has to do with having multiple teams assigned to `/content/` in the current file?

**Comment by dopplershift** [comment]
Ok, actually the persistence of the team was due to `main` being a protected branch which required a review from codeowners. Per [the docs](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team#about-code-review-assignments):
> When code owners are automatically requested for review, the team is still removed and replaced with individuals. The individual approvals don't satisfy the requirement for code owner approval in a protected branch.

So I have disabled the required review from codeowners for now. I can re-enable if we decide otherwise, but this should solve the "team stays assigned" problem.

**Review by brian-rose — APPROVED** [review]

**Review by ktyle — APPROVED** [review]

**Pull request merged**
